packages/terra-application-navigation/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Unreleased
 * Cleaned up imports in examples and test files
 * Added check to prevent rendering of Utility Button or Nav Drawer Button if they are empty.
 * Updated `keycode-js` from `v1.0.4` to `v2.0.1`
+* Update underlying structure to handle new theme variables.
```

Comment:
This snuck into a released section during a branch update

Reply:
Updated.